code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -104,13 +104,6 @@ Difficulty: Hard
 			sound_played = 'sound/effects/meteorimpact.ogg', \
 			sound_volume = 200)
 
-/mob/living/simple_animal/hostile/megafauna/bubblegum/Destroy()
-	QDEL_NULL(triple_charge)
-	QDEL_NULL(hallucination_charge)
-	QDEL_NULL(hallucination_charge_surround)
-	QDEL_NULL(blood_warp)
-	return ..()
```

Comment:
Won't this cause hard deletes? 
Just cause they'll self-delete when bubbles deletes doesn't mean the variables will null themselves

Reply:
ah right i forgot about that part